### ISSUE TITLE
SpeedGooseCacheAutoCleaner causing multitenantKey error

### ISSUE THREAD

**Issue by adjstreams** [arqo123/speedgoose#41]
I'm getting an error when trying to do a findByIdAndUpdate on a schema I'd previously forgotten to add the SpeedGooseCacheAutoCleaner plugin to, and now when I try to include it, e..g: 

```
UserSchema.plugin(SpeedGooseCacheAutoCleaner);
```

in my code when  I do a findByIdAndUpdate, I get this error (locally) - I'd never set up anything relating to multitennancy on this project before, so I don't know why it's now complaining.

```
TypeError: Cannot read properties of undefined (reading 'multitenantKey')
    at getRecordAffectedByAction ([root of project]/node_modules/speedgoose/lib/src/plugin/utils.js:22:77)
```    

The code I'm using to update:

```
    User.findByIdAndUpdate(
      { _id: req.user._id },
      {
        $set: {
          token: req.user.token,
          "section.override": valueFromUser,
        },
      },
      function (err) {
        if (err) {
          console.log(err);
          errors = [err];
          has_saved = false;
        }
      }
    );
```

**Comment by arqo123** [comment]
@adjstreams  Thanks for the feedback. It's a bug. I will fix it today.

**Comment by arqo123** [comment]
fixed in new version :) 1.2.17
